relay-replays/src/recording.rs

```diff
@@ -68,31 +68,72 @@ impl From<serde_json::Error> for ParseRecordingError {
     }
 }
 
-static STRING_STATE: Lazy<ProcessingState> = Lazy::new(|| {
-    ProcessingState::root().enter_static(
-        "",
-        Some(Cow::Owned(FieldAttrs::new().pii(Pii::True))),
-        Some(ValueType::String),
-    )
+/// Paths to fields on which datascrubbing rules should be applied.
+///
+/// This is equivalent to marking a field as `pii = true` in an `Annotated` schema.
+static PII_FIELDS: Lazy<[Vec<&str>; 2]> = Lazy::new(|| {
+    [
+        vec!["data", "payload", "description"],
+        vec!["data", "payload", "data"],
+    ]
 });
 
+/// Returns `True` if the given path should be treated as `pii = true`.
+fn scrub_at_path(path: &Vec<String>) -> bool {
+    PII_FIELDS
+        .iter()
+        .any(|p| p.iter().zip(path).all(|(k1, k2)| k1 == k2))
```

Comment:
Great catch!

Reply:
@iker-barriocanal  can you give an example what you mean with your comment?  From what I can see this should work. 

This will match of course on the shorter paths, like if the `path` will have only  2 parts provided but our `PII_FIELDS` fields have 3. 

That's what you meant? 
It's easy to unit-test this function.

Reply:
> This will match of course on the shorter paths, like if the `path` will have only  2 parts provided but our `PII_FIELDS` fields have 3. 

That's what I meant. Paths should fully match to scrub them, with no extra items on any side.